container/libcontainer/helpers.go

```diff
@@ -166,22 +167,16 @@ func scanInterfaceStats(netStatsFile string) ([]info.InterfaceStats, error) {
 
 	for scanner.Scan() {
 		line := scanner.Text()
-		if netStatLineRE.MatchString(line) {
-			line = strings.Replace(line, ":", "", -1)
-
-			i := info.InterfaceStats{}
+		line = strings.Replace(line, ":", "", -1)
 
-			_, err := fmt.Sscanf(line, "%s %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d",
-				&i.Name, &i.RxBytes, &i.RxPackets, &i.RxErrors, &i.RxDropped, &bkt, &bkt, &bkt,
-				&bkt, &i.TxBytes, &i.TxPackets, &i.TxErrors, &i.TxDropped, &bkt, &bkt, &bkt, &bkt)
+		i := info.InterfaceStats{}
 
-			if err != nil {
-				return stats, fmt.Errorf("failure opening %s: %v", netStatsFile, err)
-			}
+		numRead, _ := fmt.Sscanf(line, netstatsLine,
```

Comment:
I don't think so. we've handled the file read error so we have a string that we're just trying to match.